godot-macros/src/itest.rs

```diff
@@ -61,6 +61,7 @@ pub fn attribute_itest(input_item: venial::Item) -> ParseResult<TokenStream> {
             return bad_signature(&func);
         }
     } else {
+        // Synthetic parameter; no span.
```

Comment:
do we need this comment here? Lack of span is pretty obvious and itest is not user-facing